reference/client/get-pages.mdx

```diff
@@ -0,0 +1,366 @@
+---
+title: "getPages"
+description: "An instance method that provides data about the pages you've created within Makeswift"
+---
+
+## Options
+
+<ParamField query="limit" type="number" default={20}>
+  Number of pages to fetch. Must be between `1` and `100`.
+</ParamField>
+
+<ParamField query="after" type="string">
+  Starting `id` cursor of the last page entry that was fetched.
+</ParamField>
+
+<ParamField query="pathPrefix" type="string" default="/">
+  A filter to only retrieve pages with a pathname that begins with this value.
+</ParamField>
+
+<ParamField query="locale" type="string">
+  The site locale to fetch. 
+</ParamField>
+
+<ParamField query="sortBy" type="string" default="path"> 
+  The field to sort the result pages by. Can be any one of the following: `'title'`, `'path'`, `'description'`, `'createdAt'`, `'updatedAt'`.
+</ParamField>
+
+<ParamField query="sortDirection" type="string" default="asc">
+  The direction of how the result pages are sorted. Can be either `'asc'` or `'desc'`.
+</ParamField>
+
+<ParamField query="siteVersion" type="string" default={`Live`}>
+  Used to either retrieve the published version or working version of Makeswift
+  pages. Can either be `Live` or `Working` 
+</ParamField>
+
+<ParamField query="includeOffline" type="boolean" default={false}>
+  Whether the results should include pages that are offline in Makeswift.
+</ParamField>
+
+## Return type
+
+```ts
+type MakeswiftPage = {
+  id: string,
+  path: string,
+  title: string | null,
+  description: string | null,
+  canonicalUrl: string | null,
+  socialImageUrl: string | null,
+  sitemapPriority: number | null,
+  sitemapFrequency: string | null,
+  createdAt: string,
+  updatedAt: string,
+  publishedAt: string | null,
+  isOnline: boolean | null,
+  excludedFromSearch: boolean | null,
+  locale: string,
+  localizedVariants: {
+    locale: string,
+    path: string
+  }[]
+}
+
+// Final return type
```

Comment:
It's a bit weird that we've got a "Return type" section and then have a bunch of TypeScript and use a comment to describe the actual return type.

I'm noticing that we don't that great of a job in any of our other reference pages for the Makeswift client. Maybe we can look at other Mintlify docs examples to see how they do it.

Reply:
I went through Mintlify's showcase of customers - while I didn't exhaustively search them, I found that most docs heavily prioritize describing the payload being sent to the API, but often omit details on the returned data. One that I saw was: https://developer.ongoody.com/commerce-api/create-an-order, which takes a similar approach of showing a sample response (often times they'll have a separate schema page). Since I want to get these docs out sooner than later, I'm going to leave this as is and look through more examples to find something I'm happier with.